### ISSUE TITLE
v2.12 - upsertAsync calls are flagged as "old API"

### ISSUE THREAD

**Issue by perbergland** [meteor/meteor#12626]
When enabling warnings with WARN_WHEN_USING_OLD_API, calls to collection.upsertAsync are flagged as faulty when they in fact are correct.

```
   Calling method activities_seen.update from old API on server.
   This method will be removed, from the server, in version 3.
   Trace is below:
[object Object]
    at warnUsingOldApi (packages/mongo/collection.js:24:12)
    at Collection.update (packages/mongo/collection.js:692:5)
    at Collection.Mongo.Collection.<computed> [as update] (packages/aldeed:collection2/collection2.js:217:19)
    at Collection.upsert (packages/mongo/collection.js:808:17)
    at Collection.Mongo.Collection.<computed> [as upsertAsync] (packages/mongo/collection.js:1003:46)
```

**Comment by Grubba27** [comment]
Hello @perbergland, do you have the record of any other method that was wrongly added? 

In the tests, I only tested `insert`, `find`, and `delete`🤦

**Comment by Grubba27** [comment]
Solved in 2.13 😄
